From c80a3933635d3ed5c049fe7991ac8829d27564ea Mon Sep 17 00:00:00 2001
From: rkhan467 <Riyaz_Khan@comcast.com>
Date: Tue, 3 Sep 2019 15:58:18 +0000
Subject: [PATCH] wpeframework-plugins: Linking InjectedBundle cpp files to
 resolve undefined symbols in WPEFramework and removing module name
 declaration as its conflicting WebKitBrowser cpp(Module.cpp) file.

Signed-off-by: rkhan467 <Riyaz_Khan@comcast.com>
---
 WebKitBrowser/CMakeLists.txt          | 5 +++++
 WebKitBrowser/InjectedBundle/main.cpp | 3 ---
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/WebKitBrowser/CMakeLists.txt b/WebKitBrowser/CMakeLists.txt
index 8e98918..6509875 100644
--- a/WebKitBrowser/CMakeLists.txt
+++ b/WebKitBrowser/CMakeLists.txt
@@ -42,6 +42,11 @@ add_library(${MODULE_NAME} SHARED
     WebKitImplementation.cpp
     InjectedBundle/WhiteListedOriginDomainsList.cpp
     InjectedBundle/Utils.cpp
+    InjectedBundle/main.cpp
+    InjectedBundle/Milestone.cpp
+    InjectedBundle/NotifyWPEFramework.cpp
+    InjectedBundle/ClassDefinition.cpp
+    InjectedBundle/JavaScriptFunction.cpp
 )
 
 set_target_properties(${MODULE_NAME} PROPERTIES
diff --git a/WebKitBrowser/InjectedBundle/main.cpp b/WebKitBrowser/InjectedBundle/main.cpp
index a7b03a6..4575f93 100644
--- a/WebKitBrowser/InjectedBundle/main.cpp
+++ b/WebKitBrowser/InjectedBundle/main.cpp
@@ -263,9 +263,6 @@ static WKBundleClientV1 s_bundleClient = {
     nullptr, // didReceiveMessageToPage
 };
 
-// Declare module name for tracer.
-MODULE_NAME_DECLARATION(BUILD_REFERENCE)
-
 void WKBundleInitialize(WKBundleRef bundle, WKTypeRef)
 {
     g_Bundle = bundle;
-- 
2.23.0

